contrib/test/ci/system-packages.yml

```diff
@@ -79,6 +73,18 @@
       - gpgme-devel
   when: ansible_distribution == 'CentOS' and ansible_distribution_release == "Stream" and ansible_distribution_major_version|int > 8
 
+- name: Install optional packages For Fedora/CentOS 8
```

Comment:
the name isn't right here. also, will this work on centos 8?

Reply:
fixed, this will not work on CentOS8. There's one more stanza below which covers that part